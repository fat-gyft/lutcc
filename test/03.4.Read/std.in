wrong number
5still not ok
again
and finally:

1
2
3
4
5
