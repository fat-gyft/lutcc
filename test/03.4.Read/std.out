> Error, please type an integer.
> Error, please type an integer.
> Error, please type an integer.
> Error, please type an integer.
> Error, please type an integer.
> > > > > 10
40
90
160
250